stages/org.osbuild.pacman

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/python3
+"""
+Verify, and install pacman packages
+
+`packages` is an array of objects representing pacman packages. Each package is
+identified by its checksums. Specifically, the content hash of the package.
+
+This stage will fail if any of the packages can't be found.
+"""
+
+
+import contextlib
+import os
+import pathlib
+import shutil
+import subprocess
+import sys
+import tempfile
+from operator import itemgetter
+
+# pylint: disable=import-error
+import pyalpm
+
+from osbuild import api
+
+
+SCHEMA = """
```

Comment:
This can be removed since the code actually doesn't support the old way (like RPM does for v1 manifest).